src/dev/build/tasks/clean_tasks.js

```diff
@@ -84,3 +84,28 @@ export const CleanExtraBinScriptsTask = {
     }
   }
 };
+
+export const CleanExtraBrowsersTask = {
+  description: 'Cleaning extra browsers from platform-specific builds',
+
+  async run(config, log, build) {
+    const reporting = 'node_modules/x-pack/plugins/reporting';
+    const browsers = '{.chromium,.phantom}';
+    for (const platform of config.getPlatforms()) {
+      if (platform.isWindows()) {
+        await deleteAll(log, [
+          build.resolvePathForPlatform(platform, reporting, browsers, '*{darwin,macosx,linux}.zip'),
+        ]);
+      }
+      else if (platform.isMac()) {
+        await deleteAll(log, [
+          build.resolvePathForPlatform(platform, reporting, browsers, '*{win32,windows,linux}.zip'),
```

Comment:
The filename for the Linux phantom build is `phantomjs-2.1.1-linux-x86_64.tar.bz2`. Will this glob catch that?

Reply:
Good catch, fixed with 70754dd.